src/App.jsx

```diff
@@ -7,11 +8,28 @@ import NavBar from './components/NavBar/NavBar';
 import HdfMap from './components/HdfMap/HdfMap';
 import Members from './components/Members/Members';
 import Partners from './components/Partners/Partners';
-import Events from './components/Events/Events';
 import HdfTwitter from './components/HdfTwitter/HdfTwitter';
 import ContactForm from './components/ContactForm/ContactForm';
 import Footer from './components/Footer/Footer';
 
+function App() {
+  return (
+    <div className="App">
+      <Header />
+      <NavBar />
+      <Slider />
+      <HdfMap />
+      <Members />
+      <Partners />
+      <HdfTwitter />
+      <TwitterTweetEmbed />
+      <TwitterTimelineEmbed />
+      <ContactForm />
+      <Footer />
+    </div>
+  );
+}
+
 dotenv.config();
```

Comment:
Useless here

Reply:
We need it for the management of the order of the sections, seen in a previous PR

Reply:
What I mean is, check the whole file: there are **two** App components